tests/integration/aof.tcl

```diff
@@ -491,9 +491,11 @@ tags {"aof external:skip"} {
             set start [clock clicks -milliseconds]
             $rd debug loadaof
             $rd flush
-            after 100
-            catch {r ping} err
-            assert_match {LOADING*} $err
+            wait_for_condition 50 100 {
+                [catch {r ping} e] == 1
+            } else {
+                fail "Redis didn't enter LOADING"
+            }
```

Comment:
the previous code used to verify that we get the LOADING error, and the new one just verifies that PING fails (on whatever error).
maybe change this?
alternatively, IIUC we just wanna be sure we managed to get a response from redis while it's loading, so we can just do `[s loading] eq 1` in the `wait_for_condition` loop.

Reply:
Agree